quakemigrate/plot/event.py

```diff
@@ -148,19 +148,21 @@ def _plot_waveform_gather(ax, lut, event, idx):
     """
 
     # --- Predicted traveltimes ---
-    ttpf, ttsf = [lut.traveltime_to(phase, idx) for phase in "PS"]
-    ttp = [(event.otime + tt).datetime for tt in ttpf]
-    tts = [(event.otime + tt).datetime for tt in ttsf]
-    range_order = abs(np.argsort(np.argsort(ttp)) - len(ttp)) * 2
+    traveltimes = [lut.traveltime_to(phase, idx) for phase in lut.phases]
```

Comment:
Will have to re-visit this if we add an extra point to specify phases to be used (e.g. `onset.phases`)